generator/generator.go

```diff
@@ -360,7 +360,7 @@ func sanitizeValue(value string) string {
 		return skipHolder
 	}
 
-	name := value
+	name := strings.ReplaceAll(value, "+", "plus")
```

Comment:
So, this is a very specific change that will fix your particular problem, however, it won't necessarily be a general rule for everyone who uses this tool.  If you want to add this sort of functionality, I would much prefer seeing a more generic way of changing symbols during processing.  If you can come up with that, then I'd be glad to review it, otherwise, I'll need to come up with a solution to the problem, and that will take some time.

Also, I would need to see tests that validate this does what you want it to do.  Not sure why circle decided to not run any tests on your fork, but it should have.